### PR TITLE
Luxianze/side nav bar

### PR DESCRIPTION
**Pull request by LUXIANZE**

### CODE REVIEW

File: src/main/webapp/app/entities/event/event-update.tsx [hunk 1]
```diff
@@ -160,7 +161,13 @@ export class EventUpdate extends React.Component<IEventUpdateProps, IEventUpdate
                 </AvGroup>
                 <AvGroup>
                   <Label id="requiredTransportLabel" check>
-                    <AvInput id="event-requiredTransport" type="checkbox" className="form-control" name="requiredTransport" />
+                    <AvInput
+                      id="event-requiredTransport"
+                      type="checkbox"
+                      className="form-control"
+                      name="requiredTransport"
+                      style={{ height: '20px', marginTop: '0px', marginLeft: '150px' }}
```
Comment (HowardYaw) [inline src/main/webapp/app/entities/event/event-update.tsx]: suggest put it as a checkbox css style
so we can reuse this standard style for all checkbox

File: src/main/webapp/i18n/en/event.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
       "home": {
         "title": "Events",
         "createLabel": "Create a new Event",
-        "createOrEditLabel": "Create or edit a Event",
+        "createOrEditLabel": "Create New Event",
```
Comment (HowardYaw) [inline src/main/webapp/i18n/en/event.json]: are you sure key **createOrEditLabel** value is **Create New Event**?

**Review by kachun333 — DISMISSED** [review]

**Review by HowardYaw — DISMISSED** [review]

File: src/main/webapp/i18n/en/event.json (hunk 2 above)
Comment (LUXIANZE) [reply src/main/webapp/i18n/en/event.json]: When edit, i will make it the event name. When create will appear as create new event

File: src/main/webapp/app/entities/event/event-update.tsx (hunk 1 above)
Comment (LUXIANZE) [reply src/main/webapp/app/entities/event/event-update.tsx]: Ok will do

File: src/main/webapp/i18n/en/event.json (hunk 2 above)
Comment (HowardYaw) [reply src/main/webapp/i18n/en/event.json]: @LUXIANZE 
How to render based on create and update?
based on the event-update i saw this only
image

Comment (LUXIANZE) [reply src/main/webapp/i18n/en/event.json]: will update, miss this part

Comment (LUXIANZE) [reply src/main/webapp/i18n/en/event.json]: decided to revert to original since the name will be displayed in form.

**Review by HowardYaw — APPROVED** [review]

**Review by CarlosSia — APPROVED** [review]

**Pull request merged**